src/modules/components/proposal/proposalVoting/proposalVotingStage/proposalVotingStage.tsx

```diff
@@ -33,81 +32,83 @@ export interface IProposalVotingStageProps extends ComponentProps<'div'> {
      * Defines if the proposal has multiple stages or not.
      */
     isMultiStage?: boolean;
+    /**
+     * List of plugin addresses of bodies.
+     */
+    bodyList?: string[];
 }
 
 export const ProposalVotingStage: React.FC<IProposalVotingStageProps> = (props) => {
-    const { name, status, startDate, endDate, index, children, isMultiStage, className, ...otherProps } = props;
+    const { name, status, startDate, endDate, index, children, isMultiStage, className, bodyList, ...otherProps } =
+        props;
 
     const { copy } = useGukModulesContext();
 
-    const futureStatuses = [ProposalVotingStatus.PENDING, ProposalVotingStatus.UNREACHED];
-    const stateActiveTab = futureStatuses.includes(status) ? ProposalVotingTab.DETAILS : ProposalVotingTab.BREAKDOWN;
-
-    const [activeTab, setActiveTab] = useState<string | undefined>(stateActiveTab);
-
-    // Update active tab when stage status changes (e.g from PENDING to UNREACHED)
-    useEffect(() => setActiveTab(stateActiveTab), [stateActiveTab]);
+    const [activeBody, setActiveBody] = useState<string | undefined>(
+        bodyList && bodyList.length === 1 ? bodyList[0] : undefined,
+    );
 
     const accordionContentRef = useRef<HTMLDivElement>(null);
 
-    const contextValues = useMemo(() => ({ startDate, endDate }), [startDate, endDate]);
-
+    const contextValues = useMemo(
+        () => ({ startDate, endDate, bodyList, activeBody, setActiveBody }),
+        [startDate, endDate, bodyList, activeBody],
+    );
     invariant(
         !isMultiStage || index != null,
         'ProposalVotingStage: component must be used inside a ProposalVotingContainer to work properly.',
     );
 
+    const renderContent = () => {
+        if (activeBody || !bodyList) {
+            // Render children
+            return children;
+        }
+        // Render ProposalVotingBodySummary when activeBody is null and bodyList has more than one element
+        return <ProposalVotingBodySummary>{children}</ProposalVotingBodySummary>;
```

Comment:
Shouldn't we just always render the children? The body-summary is already rendered by the parent component and it's hidden if activeBody is null